spikemetrics/metrics.py

```diff
@@ -876,3 +937,29 @@ def get_unit_pcs(these_pc_features, index_mask, channel_mask):
     unit_PCs = unit_PCs[:, :, channel_mask]
 
     return unit_PCs
+
+def find_neighboring_channels(peak_channel, channel_list, num_channels_to_compare, channel_locations):
+    """
+    Finds k nearest channels to the peak channel of a unit
+
+    Parameters
+    ----------
+    peak_channel: int
+        id of channel with largest waveform amplitude
+    channel_list: array_like
+        ids of channels being considered
+    num_channels_to_compare: int
+        number of nearest channels to return
+    channel_locations: array_like, (n_channels, 2)
+        x,y coordinates of the channels in channel_list
+
+    Returns
+    -------
+    neighboring_channels: array_like
+        id of k channels that neighbor peak channel (including the peak channel itself)
+    """
+    neighboring_channels_inds = NearestNeighbors(n_neighbors=num_channels_to_compare,
+                                algorithm='auto').fit(channel_locations).kneighbors(channel_locations[[i==peak_channel for i in channel_list]],
+                                                                    return_distance=False)[0]
```

Comment:
@khl02007 wouldn't it be easier to just compute distances to the other channels and argsort them to get the closest channels?

Reply:
Yes I agree. Is there a way for me to change this one line or do I have to submit another pull request?

Reply:
You can just make the change, commit and push on your fork and it will automatically update

Reply:
I see, just did that.